README.md

```diff
@@ -42,6 +42,5 @@ The Maven Build Scan® quickstart project is open-source software released under
 [apache-license]: https://www.apache.org/licenses/LICENSE-2.0.html
 [manual]: https://docs.gradle.com/enterprise/maven-extension
 [gradle.com]: https://www.gradle.com
-[terms-of-service]: https://gradle.com/terms-of-service
```

Comment:
Shouldn't this be replaced with https://gradle.com/legal/terms-of-use?

Reply:
There is a redirect. We use the same url in other samples.